Makefile

```diff
@@ -100,6 +100,9 @@ test_client: build_client_source
 build_client_container: test_client build_client_source
 	${DC_BUILD} build reviewer_client
 
+push_client_container: build_client_container
```

Comment:
this is not used in any other targets, so the build doesn't trigger it

Reply:
... unless you explicitly called this from the build script - maybe you forgot to commit a file?